src/Avalonia.Themes.Fluent/TextBox.xaml

```diff
@@ -46,57 +54,60 @@
-                      </MultiBinding>
-                  </TextBlock.IsVisible>
-                </TextBlock>
-
-                <DataValidationErrors>
-                  <ScrollViewer HorizontalScrollBarVisibility="{TemplateBinding (ScrollViewer.HorizontalScrollBarVisibility)}"
-                                VerticalScrollBarVisibility="{TemplateBinding (ScrollViewer.VerticalScrollBarVisibility)}">
-
-                    <Panel>
-                      <TextBlock Name="watermark"
-                               Opacity="0.5"
-                               Text="{TemplateBinding Watermark}"
-                               TextAlignment="{TemplateBinding TextAlignment}"
-                               TextWrapping="{TemplateBinding TextWrapping}"
-                               IsVisible="{TemplateBinding Text, Converter={x:Static StringConverters.IsNullOrEmpty}}"
-                               HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}" 
-                               VerticalAlignment="{TemplateBinding VerticalContentAlignment}"/>
-                      <!-- TODO eliminate this margin... text layout issue? -->
-                      <TextPresenter Name="PART_TextPresenter"
-                                   Margin="0 1 0 0"
-                                   Text="{TemplateBinding Text, Mode=TwoWay}"
-                                   CaretIndex="{TemplateBinding CaretIndex}"
-                                   SelectionStart="{TemplateBinding SelectionStart}"
-                                   SelectionEnd="{TemplateBinding SelectionEnd}"
-                                   TextAlignment="{TemplateBinding TextAlignment}"
-                                   TextWrapping="{TemplateBinding TextWrapping}"
-                                   PasswordChar="{TemplateBinding PasswordChar}"
-                                   SelectionBrush="{TemplateBinding SelectionBrush}"
-                                   SelectionForegroundBrush="{TemplateBinding SelectionForegroundBrush}"
-                                   CaretBrush="{TemplateBinding CaretBrush}"
-                                   HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}" 
-                                   VerticalAlignment="{TemplateBinding VerticalContentAlignment}"/> 
-                    </Panel>
-                  </ScrollViewer>
-                </DataValidationErrors>
-              </DockPanel>
+              <Grid ColumnDefinitions="Auto,*,Auto" >
+                <ContentPresenter Grid.Column="0" Grid.ColumnSpan="1" Content="{TemplateBinding InnerLeftContent}"/>
+                <DockPanel Grid.Column="1" Grid.ColumnSpan="1" Margin="{TemplateBinding Padding}">
+                  <TextBlock Name="floatingWatermark"
+                            Foreground="{DynamicResource SystemAccentColor}"
+                            FontSize="{DynamicResource FontSizeSmall}"
```

Comment:
FontSizeSmall is missed in Fluent theme. You need to replace it with ControlContentThemeFontSize (there is not "small" equivalent) or restore "TemplateBinding FontSize" as it was before that PR. 
```suggestion
                            FontSize="{TemplateBinding FontSize}"
```

Reply:
Fixed in https://github.com/AvaloniaUI/Avalonia/pull/4487